aware-core/src/main/java/com/aware/Applications.java

```diff
@@ -85,6 +90,55 @@ public class Applications extends AccessibilityService {
 
     public String AUTHORITY = "";
 
+    // Screentext variables
+    private String currScreenText = "";
+
+    ArrayList<ContentValues> contentBuffer = new ArrayList<ContentValues>();
+
+    ArrayList<Integer> textBuffer = new ArrayList<Integer>();
+
+    int TEXT_BUFFER_LIMIT = 100;
+
+    int mDebugDepth = 0;
```

Comment:
Does this still need to be included?

Reply:
Do you mean TEXT_BUFFER_LIMIT?

Reply:
Sorry, mDebugDepth

Reply:
Well, it seems to be used when we follow the tree structure of the widgets on the screen.